sapphire/tests/transformations/test_celestial.py

```diff
@@ -1,12 +1,22 @@
 import calendar
 import datetime
 import unittest
+
 from math import pi
 
+
 import numpy as np
 
 from sapphire.transformations import base, celestial, clock
 
+# This is to switch off tests in case astropy is not present
+try:
+    import astropy
+    has_astropy = True
+    astropy.__version__  # Make flake8 shut up about astropy unused
```

Comment:
Use `# noqa` after the import astropy instead of this.

http://flake8.pycqa.org/en/latest/user/violations.html?highlight=noqa#in-line-ignoring-errors